src/controllers/auth.controller.js

```diff
@@ -0,0 +1,66 @@
+const authService = require('../services/auth.service');
+
+const register = async (req, res) => {
+  const userBody = req.body;
+  const newUser = await authService.register(userBody);
+
+  res.status(201).send(newUser);
+};
+
+const activate = async (req, res) => {
+  const { activationToken } = req.params;
+  const user = await authService.activate(activationToken);
+
+  res.send(user);
```

Comment:
The `activate` function should redirect the user to the profile page after activation, as specified in the task description. Consider adding a redirect after the activation process is successful.